Source/FlowEditor/Private/Graph/FlowGraph.cpp

```diff
@@ -31,7 +31,13 @@ UFlowGraph::UFlowGraph(const FObjectInitializer& ObjectInitializer)
 
 UEdGraph* UFlowGraph::CreateGraph(UFlowAsset* InFlowAsset)
 {
-	UEdGraph* NewGraph = CastChecked<UFlowGraph>(FBlueprintEditorUtils::CreateNewGraph(InFlowAsset, NAME_None, StaticClass(), UFlowGraphSchema::StaticClass()));
+	return CreateGraph(InFlowAsset, UFlowGraphSchema::StaticClass());
+}
+
+UEdGraph* UFlowGraph::CreateGraph(UFlowAsset* InFlowAsset, TSubclassOf<UFlowGraphSchema> FlowSchema)
```

Comment:
We use our own Schema (subclass) so we needed a CreateGraph function that accepted a user-specified schema.

We may want to remove the other version of this function (w/o the explicit schema param), but I didn't make that change in this changelist.